src/common-control/common-control-styles.html

```diff
@@ -0,0 +1,8 @@
+<!-- shared styles for all custom components -->
+<!-- Styling done the Polymer way https://youtu.be/IbOaJwqLgog?t=8m11s -->

```

Comment:
Good ~15minute about what additional functionality you get by using css support offered by polymer. https://www.youtube.com/watch?v=IbOaJwqLgog